.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "bracketSameLine": true,
+  "printWidth": 100,
+  "singleQuote": true
```

Comment:
I took settings from sonarcloud-webapp repo, we can tune them to also match SQ repo if needed